data/categories.txt

```diff
@@ -576,6 +576,7 @@ he:כספומט
 sk:3Bankomat
 sw:Benki|fedha
 fa:خود پرداز
+hi:एटीएम|स्वचालित गणक मशीन (एटीएम)
```

Comment:
1. Just to make sure: the first word matches strings.txt as it mentioned in the comment above, right? (`# First keyword should match [atm] definition in strings.txt!`)
2. categories.txt should have texts that can be typed by users. Are you sure that someone can type the second phrase here? If it can be typed, then you may add a number prefix before the longer word (usually 3, 4, or 5) to display the whole phrase as a suggestion, so instead of typing users can select the suggestion and it will be pasted into the search box.

Reply:
ad 1:  I'm pretty sure I considered it back than, but it's a long time ago already. So I will check again if the first word matches the word in strings.txt.
ad 2: I can add numbers (between 3-5) for single words depending on wordlength and my feeling of importance. For phrases idk. That could be left to natives, wdyt?